documentation/release_notes.markdown

```diff
@@ -181,15 +181,19 @@ derive data from a PuppetDB export tarball.
 
 * Add a `--url-prefix` option to the import and export commands in the PuppetDB CLI  ([PDB-1068](https://tickets.puppetlabs.com/browse/PDB-1068))
 
-    This allows import and export to work against a PuppetDB configured with a url prefix. 
+    This allows import and export to work against a PuppetDB configured with a url prefix.
 
 * Add the `[puppetdb] disable-update-checking` configuration key ([PDB-158](https://tickets.puppetlabs.com/browse/PDB-158))
 
-    The value defaults to false; see the [configuration](./configure.html) documentation for more details. 
+    The value defaults to false; see the [configuration](./configure.html) documentation for more details.
+
+* Added ability to expand child data for reports/facts & catalogs endpoints.
+
+* The `reports` database table was using the hash string as its primary key, we know have switched to using a smaller bigint primary key for that table, which should result in faster joins in most cases, and hopefully smaller foreign key indexes sizes for tables relating to `reports`. ([PDB-1218](https://tickets.puppetlabs.com/browse/PDB-1218))
```

Comment:
Lol, umm, yeah this point doesn't belong in this patch. But thanks for the review :-/.